jdi-light/src/main/java/com/epam/jdi/light/common/JDILocator.java

```diff
@@ -21,22 +23,24 @@
  */
 public class JDILocator {
     public JDILocator() {}
+    public JDILocator(JDIBase element) { this.element = element; }
     public JDILocator copy() {
         JDILocator locator = new JDILocator();
-        locator.locatorType = locatorType;
         locator.byLocator = byLocator;
         locator.isRoot = isRoot;
         locator.element = element;
+        locator.frames = frames;
         return locator;
     }
 
-    private LocatorType locatorType = DEFAULT;
     private By byLocator;
+    private List<By> frames;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4522228)